sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveDDLCommandSuite.scala

```diff
@@ -556,4 +558,32 @@ class HiveDDLCommandSuite extends PlanTest {
     assert(partition2.get.apply("c") == "1" && partition2.get.apply("d") == "2")
   }
 
+  test("Test default fileformat") {
```

Comment:
Please update the test case name to `Test the default fileformat for Hive-serde tables`